src/buffer/out/textBuffer.cpp

```diff
@@ -1359,9 +1359,15 @@ const til::point TextBuffer::GetGlyphEnd(const til::point pos) const
 bool TextBuffer::MoveToNextGlyph(til::point& pos, bool allowBottomExclusive) const
 {
     COORD resultPos = pos;
+    const auto bufferSize = GetSize();

```

Comment:
is this the fix for the crash?

Reply:
Yup. In general, I thought it was only possible for `_end` to be at `EndExclusive()`, so `allowBottomExclusive` is used to track if we are working with `_end`. That definitely wasn't the case because you can end up with `_start` at `EndExclusive()` by doing the repro steps.

So this check for `EndExclusive()` is basically thrown around everywhere to ensure we don't crash on an `IncrementInBounds` or `DecrementInBounds` check (which checks if the COORD is in the buffer, and `EndExclusive` isn't)